integration-test/src/it/circular-dependencies-test-second/test.properties

```diff
@@ -17,5 +17,5 @@
 restURL=http://127.0.0.1:8089
 versionIncrementalSuffix=redhat
 dependencySource=rest
-groovyScripts=file://${basedir}/src/it/circular-dependencies-test-second/Sample.groovy
```

Comment:
The `file://` as I thought we discussed previously is an acceptable standard so I don't see why this needs changing?

Reply:
URLs beginning with `file://` are definitely incorrect, but I can revert the changes to the properties files and try to fix up the invalid URL in the code instead. I can add a test case to show the failure.

Note that such a URL parses, so in that sense it is _valid_, but it's _semantically_ incorrect in that it expects an `authority` (hostname) to come after two slashes, which is true in the case of a network URL, but a file URL should have no `authority`

Reply:
Lets discuss offline.

Reply:
All I know is that `file:` with no slashes appeared to work on all OSes via the CI. With two or three slashes, there will need to be code to fix up the URI on Windows.